landlab/components/lateral_erosion/lateral_erosion.py

```diff
@@ -0,0 +1,631 @@
+        max_slopes[:] = max_slopes.clip(0)
+        for i in dwnst_nodes:
+            # calc deposition and erosion
+            dep = alph * qs_in[i] / da[i]
+            ero = -Kv[i] * da[i] ** (0.5) * max_slopes[i]
+            dzver[i] = dep + ero
+            # potential lateral erosion initially set to 0
+            petlat = 0.0
+            # water depth in meters, needed for lateral erosion calc
+            wd = wid_coeff * (da[i] * runoffms) ** wid_exp
+
+            # Choose lateral node for node i. If node i flows downstream, continue.
+            # if node i is the first cell at the top of the drainage network, don't go
+            # into this loop because in this case, node i won't have a "donor" node
+            if i in flowdirs:
+                # node_finder picks the lateral node to erode based on angle
+                # between segments between three nodes
+                [lat_node, inv_rad_curv] = node_finder(grid, i, flowdirs, da)
+                # node_finder returns the lateral node ID and the radius of curvature
+                lat_nodes[i] = lat_node
+                # if the lateral node is not 0 or -1 continue. lateral node may be
+                # 0 or -1 if a boundary node was chosen as a lateral node. then
+                # radius of curavature is also 0 so there is no lateral erosion
+                if lat_node > 0:
+                    # if the elevation of the lateral node is higher than primary node,
+                    # calculate a new potential lateral erosion (L/T), which is negative
+                    if z[lat_node] > z[i]:
+                        petlat = -Kl[i] * da[i] * max_slopes[i] * inv_rad_curv
+                        # the calculated potential lateral erosion is mutiplied by the length of the node
+                        # and the bank height, then added to an array, vol_lat_dt, for volume eroded
+                        # laterally  *per timestep* at each node. This vol_lat_dt is reset to zero for
+                        # each timestep loop. vol_lat_dt is added to itself in case more than one primary
+                        # nodes are laterally eroding this lat_node
+                        # volume of lateral erosion per timestep
+                        vol_lat_dt[lat_node] += abs(petlat) * grid.dx * wd
+
+            # send sediment downstream. sediment eroded from vertical incision
+            # and lateral erosion is sent downstream
+            qs_in[flowdirs[i]] += (
+                qs_in[i] - (dzver[i] * grid.dx ** 2) - (petlat * grid.dx * wd)
+            )  # qsin to next node
+        qs[:] = qs_in - (dzver * grid.dx ** 2)
+        dzdt[:] = dzver * dt
+        vol_lat[:] += vol_lat_dt * dt
+        # this loop determines if enough lateral erosion has happened to change
+        # the height of the neighbor node.
+        for i in dwnst_nodes:
+            lat_node = lat_nodes[i]
+            wd = 0.4 * (da[i] * runoffms) ** 0.35
```

Comment:
I think this is a place where you mean to use the properties defined in the init rather than hard coding the values.

Reply:
yes, I missed these places. changed to use the properties defined.